.github/workflows/integration-testing.yaml

```diff
@@ -1,7 +1,8 @@
 name: Run Integration Tests
 on:
-  workflow_dispatch: {}
-  merge_group: {}
+  push:
+    branches:
+      - master
```

Comment:
This will only run after a merge has completed as written.  That is, the tests will only highlight failures on code that is already in master...  The original trigger intentions were:
- workflow dispatch to allow engineers to trigger on demand tests on a branch
- merge_group to run the tests after merge approval, but before merge completes.

Reply:
ah yes, that is a better flow, Ill revert it back

Reply:
Great - note that you will need to enable merge queues on this repo.  I can't remember if that has been done already @edamboritz